tensorflow_asr/models/layers/depthwise_conv1d.py

```diff
@@ -0,0 +1,214 @@
+"""
```

Comment:
@andreselizondo-adestech Can you format this file with pep8 style as in the `setup.cfg` file?

Reply:
Done